Makefile

```diff
@@ -103,6 +106,9 @@ signed_xpi: addon
 	./node_modules/.bin/web-ext sign --api-key=${AMO_USER} --api-secret=${AMO_SECRET} --source-dir addon/webextension/
 	mv web-ext-artifacts/*.xpi build/pageshot.xpi
 
+addon_locales: $(l10n_dest)
```

Comment:
Upon further reflection, I guess an empty dependencies list is the right thing here, given that we don't have recipes for how to generate those individual files.  Either way I believe the .json files get rebuilt on every make run because we're not telling make enough for it to know when a rebuild is necessary.